src/v2/view-builder/views/password/EnrollAuthenticatorPasswordView.js

```diff
@@ -87,11 +89,13 @@ export default BaseAuthenticatorView.extend({
       local,
       validate () {
         if (this.get('credentials.passcode') !== this.get('confirmPassword') &&
-          this.get('credential.value') !== this.get('confirmPassword')) {
+            this.get('credential.value') !== this.get('confirmPassword')) {
```

Comment:
`nit:` how about this? This just avoids an extra errors var:
```
return (this.get('credentials.passcode') !== this.get('confirmPassword')
  && this.get('credential.value') !== this.get('confirmPassword')
  ? { 'confirmPassword': loc('password.error.match', 'login') }
  : null );
```

Reply:
or we can just avoid the `else` and `return null;` after if conditional.

Reply:
> or we can just avoid the else and return null; after if conditional.

that works. I'm not consistent at this style but usually I like explicit `if/else`